services/QuillLMS/client/app/bundles/Staff/components/comprehension/activitySessions/promptTable.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import stripHtml from "string-strip-html";
+
+import { DataTable, Spinner } from '../../../../Shared/index';
+import { PROMPT_ATTEMPTS_FEEDBACK_LABELS, PROMPT_HEADER_LABELS } from '../../../../../constants/comprehension';
+import { ActivityInterface, PromptInterface } from "../../../interfaces/comprehensionInterfaces";
+
+interface PromptTableProps {
+  activity: ActivityInterface;
+  prompt: PromptInterface;
+  showHeader?: boolean;
+}
+const PromptTable = ({ activity, prompt, showHeader }: PromptTableProps) => {
+
+
+  function formatFirstTableData(prompt: any) {
+    const { attempts } = prompt;
+    const keys = attempts && Object.keys(attempts);
+    let completed: boolean;
+    if(keys.length === 5) {
+      completed = true;
+    } else {
+      const usedAttempts = [];
+      keys.forEach(key => {
+        attempts[key].forEach((attempt: any) => {
+          if(attempt.used) {
+            usedAttempts.push(attempt);
+          }
+        });
+      });
+      completed = !!usedAttempts.some((attempt) => attempt.optimal);
+    }
+    return {
+      attemptsLabel: 'Attempts',
+      attemptsValue: keys.length,
+      completedLabel: 'Completed',
+      completedValue: completed ? 'True' : 'False'
+    }
+  }
+
+  function formatFeedbackData(prompt: any) {
+    const { attempts, prompt_id } = prompt;
+    const { prompts } = activity;
+    const matchedPrompt = prompts.filter(prompt => prompt.id === prompt_id)[0];
```

Comment:
We should name the inner variable here (`prompt`) something other than the function argument `prompt`.